packages/kcmsf/src/utils/notifyError.ts

```diff
@@ -0,0 +1,21 @@
+import { notifications } from "@mantine/notifications";
+
+export const errorMessages = {
+  shortTeamName: "チーム名が短すぎます",
+  shortMemberName: "メンバーの名前が短すぎます",
+  invalidCategory: "部門が不正です",
+  invalidRobotCategory:
+    "ロボットのカテゴリーは車輪型または歩行型にしてください",
+  shortClubName: "クラブ名が短すぎます",
+  duplicateTeamName: "チーム名が重複しています",
+} as const;
+
+export type ErrorMessages = keyof typeof errorMessages;
```

Comment:
キーなのでメッセージではありません。加えてコレクションではないため、単数形が適切です。
```suggestion
export type ErrorKey = keyof typeof errorMessages;
```